src/queue.jl

```diff
@@ -4,6 +4,11 @@ type Queue{T}
     store::Deque{T}
 end
 
+"""
+    Queue(ty{T}[, blksize::Integer])
```

Comment:
Not sure what `ty{T}` is here. I would just write it as `T`, and mention the meaning of this argument in the description.